components/Market/utils.tsx

```
@@ -0,0 +1,202 @@
+export function shouldDisplayDiffTooltopEver(percentDiff: number): boolean {
+  return isFinite(percentDiff);
+}
+
+/**
+ * Returns the CSS class that should be used when displaying the provided percent difference.
+ *
+ * @param percentDiff The percentage, expressed as a number where 100 means 100%.
+ * @returns The class to use for the label.
+ */
+export function getPercentDiffLabelClass(percentDiff: number): string {
+  if (isDiffNegligible(percentDiff) || !isFinite(percentDiff)) {
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)